### PR TITLE
Update aspnet50/aspnetcore50 => dnx50/dnxcore50.

### PR DESCRIPTION
**Pull request by NTaylorMullen**
Prepping to be merged once ready.

### CODE REVIEW

File: build.cmd [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@echo off
+﻿@echo off
```
Comment (dougbu) [inline build.cmd]: what is the noise on first and last lines of many files?

File: build.sh [hunk 2]
```diff
@@ -27,12 +27,13 @@ if test ! -d packages/KoreBuild; then
     mono .nuget/nuget.exe install Sake -version 0.2 -o packages -ExcludeVersion
 fi
 
-if ! type k > /dev/null 2>&1; then
-    source packages/KoreBuild/build/kvm.sh
+type dnvm > /dev/null 2>&1; then
```
Comment (pranavkm) [inline build.sh]: Looks like this is missing the `if`. Does this still work?

Comment (NTaylorMullen) [inline build.sh]: :+1:

File: build.sh [hunk 3]
```diff
@@ -27,12 +27,13 @@ if test ! -d packages/KoreBuild; then
     mono .nuget/nuget.exe install Sake -version 0.2 -o packages -ExcludeVersion
 fi
 
-if ! type k > /dev/null 2>&1; then
-    source packages/KoreBuild/build/kvm.sh
+if ! type dnvm > /dev/null 2>&1; then
```
Comment (pranavkm) [inline build.sh]: While you're updating all the repos, also update the build files in Universe\build-template. @troydai's updated it recently, but the contents of this file look more correct

**Pull request merged**